src/linear_algebra/eigenspace.lean

```diff
@@ -56,13 +56,20 @@ def eigenspace (f : End R M) (μ : R) : submodule R M :=
 
 /-- A nonzero element of an eigenspace is an eigenvector. (Def 5.7 of [axler2015]) -/
 def has_eigenvector (f : End R M) (μ : R) (x : M) : Prop :=
-x ≠ 0 ∧ x ∈ eigenspace f μ
+x ∈ eigenspace f μ ∧ x ≠ 0
```

Comment:
I'm curious about the decision to switch this around. (Not that I have a reason to prefer it the old way.)

Reply:
It ... appears to be solely so that I can write 
```exact has_eigenvalue_of_has_eigenvector (exists_mem_ne_zero_of_ne_bot nu).some_spec```
below, without having to flip over what I pull out of `exists_mem_ne_zero_of_ne_bot`.

One the one hand: why bother making this small change that affects several files? On the other hand: the changes required are all neutral, so why not?